### PR TITLE
Add license

### PR DESCRIPTION
**Pull request by kungyao**

### CODE REVIEW

File: AUTHORS [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# This is the list of ths significant contributors.
+
+Kung-Yao Hsieh
+Shang-Chen LIN
+Yi-Hang Xie
```
Comment (john0312) [inline AUTHORS:5]: Please add new line at the end.

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,28 @@
+BSD 3-Clause License
+
+Copyright (c) 2021, Group5 Authors. All rights reserved.
```
Comment (john0312) [inline LICENSE:3]: Please use the project name.

File: AUTHORS (hunk 1 above)
Comment (kungyao) [reply AUTHORS]: Remove author file and use license file only instead.

File: LICENSE (hunk 2 above)
Comment (kungyao) [reply LICENSE]: Use contributor name

**Review by john0312 — APPROVED** [review]

**Pull request merged**